Contact form section

lily-coder opened the pull request:
This is the contact form section
I have worked on the following:
- The footer section
- The form section

emmanuellekamwa reviewed (changes requested):
**STATUS: CHANGES REQUESTED 🛑🛠**
Hello @lily-coder 👋🏽
Good job so far!
You have done most of the hard work but you still need to work on some issues to go to the next project, but you are almost there! 💪🏽

## Required Changes ♻️
_Check the comments under the review._ 👇🏽👀
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lily-coder commented on index.html:
```diff
@@ -1,133 +1,216 @@
+          maxlength="30"
+          placeholder="Text Input"
+          required
+        />
```
The webpage I am creating is the mobile version and it does not have two separate name fields, but one. The one you are comparing with is the desktop version, and we have not yet started to work on the desktop version. I kindly seek clarification on this part

emmanuellekamwa replied:
It was not precise on my side that you had to do only the mobile version of the page.

Emmyn5600 commented on index.html:
```diff
@@ -1,133 +1,216 @@
+          placeholder="Text Input"
+          required
+        />
+        <input
+          type="email"
+          name="user_email"
+          id="email"
+          placeholder="jeremy@gmail.com"
+          pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$"
+          required
+        />
+        <textarea
+          name="user_message"
+          id="message"
+          cols="30"
+          rows="10"
+          maxlength="500"
+          placeholder="Enter text here"
+          required
+        >
+        </textarea>
+        <button type="submit">See Project</button>
```
- [ ]  Good work to validate email very well with a pattern, however, try to fix your text-area field because it allows users to send an empty message, please validate it as required so that no one should send an empty message.

- [ ] Nice you are following the Figma, but I recommend you to change your button to Get It to touch instead of seeing Project (See project).

Emmyn5600 commented on index.html:
```diff
@@ -1,133 +1,216 @@
+          with it
+        </p>
+        <input
+          type="text"
+          name="user_name"
+          id="name"
+          maxlength="30"
+          placeholder="Text Input"
+          required
```
- [ ] Kindly, change your placeholder message to your full names instead of Text Input because we need to receive the user's name

Emmyn5600 reviewed (changes requested):
## STATUS: Changes Requested :recycle:

Hi @lily-coder 

Good job so far!:clap::clap:
You have done well in implementing the changes requested by the previous reviewer:clap:, However, there are some issues that you still need to work on to go to the next project but you are almost there!:muscle:

May I highlight the following:
- The repo looks professional:heavy_check_mark:
- Commit messages are good and descriptive:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have a descriptive PR:heavy_check_mark:


##  Required Changes:writing_hand:

**_Check the comments under the review._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

lily-coder commented:
I have implemented all changes as follows:

-I have fixed the text area, it cannot allow an empty message
-I have changed the button from "See Project" to "Get In Touch"
-I have changed the placeholder message for the name input from "Text Input" to "Your Full Names"

carlos-ssh commented on index.html:
```diff
@@ -1,133 +1,215 @@
-            </div>
-            <div class="story-btn1 btn1 bgb white">See Project</div>
+        <div class="story-lang1">
+          <p class="white fcss">css</p>
+          <hr />
+          <p class="white fhtml">html</p>
+          <hr />
+          <p class="white fbootstrap">bootstrap</p>
+          <hr />
+          <p class="white fruby">ruby</p>
```
- Kindly give the correct space between elements of your div.

carlos-ssh commented on index.html:
```diff
@@ -1,133 +1,215 @@
+        <button type="submit">Get In Touch</button>
+      </form>
+    </section>
+  <footer>
+    <hr />
+    <div class="footer-icons">
+      <i class="fa fa-github white"></i>
+      <i class="fa fa-linkedin white"></i>
+      <i class="fa fa-angellist white"></i>
+      <i class="fa fa-twitter white"></i>
+      <i class="fa fa-medium white"></i>
     </div>
+    <p class=" footer-email">emailhere@email.com</p>
+  </footer>
```
- Kindly fix the bottom of your page, it has a lot of in blank space.
![Screen Shot 2021-07-14 at 05 28 07](https://user-images.githubusercontent.com/22258274/125607577-0be2d769-0ca0-4be7-8fda-b2d4b6a0a22e.png)

carlos-ssh reviewed (changes requested):
Hi @lily-coder, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lily-coder commented:
In this project, I am working on the mobile view version of the webpage. I have implemented the changes above as follows:
-I have given the correct space between elements of my div
-I have fixed the bottom of my page and it does not have a lot of blank space in mobile view

nidalaa commented:
Hello  @lily-coder 👋,

This is an additional review as a result of the second opinion request. Indeed your first review was asking for unnecessary changes.

Because you have implemented all of the requested changes this project is approved now.

**You will see changes in your Dashboard in ~1h.**
